packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -528,9 +564,13 @@ export const Webchat = forwardRef((props, ref) => {
             />
           )}
           {persistentMenuOptions && (
-            <ConditionalAnimation>
-              <PersistentMenu onClick={handleMenu} />
-            </ConditionalAnimation>
+            <FeaturesWrapper>
```

Comment:
what's `FeaturesWrapper`, where is defined?

Reply:
I have used the one that is already defined, just to center the menu button. If not, when there is a lot of text in the input container the menu button remains on the top whereas the other buttons are centered. Should I create a new div instead of using FeaturesWrapper?